### ISSUE TITLE
decode_responses

### ISSUE THREAD

**Issue by cgloeckner** [bottlepy/bottle-redis#7]
Hi, it would be great if the plugin ctor could support `decode_responses` in order to convert all byte-strings to unicode-strings automatically, which is very useful when using Python 3.x.

Maybe the ctor could be made even more flexible using `*args` and `**kwargs`, e.g.
```python
def __init__(self, keyword="rdb", *args, **kwargs):
	self.keyword = keyword
	self.redisdb = redis.ConnectionPool(*args, **kwargs)
```

**Comment by avelino** [comment]
You can implement and send via PR?

**Comment by cgloeckner** [comment]
I'll try.

**Comment by avelino** [comment]
Fixed in #8
